core/src/main/java/apoc/trigger/Trigger.java

```diff
@@ -109,26 +79,75 @@ public Stream<TriggerInfo> list() {
                 );
     }
 
+    // TODO - change with @SystemOnlyProcedure
+    @SystemProcedure
+    @Procedure(mode = Mode.WRITE)
+    @Description("CALL apoc.trigger.install(databaseName, name, statement, selector, config) | add a trigger kernelTransaction under a name, in the kernelTransaction you can use $createdNodes, $deletedNodes etc., the selector is {phase:'before/after/rollback/afterAsync'} returns previous and new trigger information. Takes in an optional configuration.")
+    public Stream<TriggerInfo> install(@Name("databaseName") String databaseName, @Name("name") String name, @Name("kernelTransaction") String statement, @Name(value = "selector")  Map<String,Object> selector, @Name(value = "config", defaultValue = "{}") Map<String,Object> config) {
+        Util.validateQuery(ApocConfig.apocConfig().getDatabase(databaseName), statement);
+
+        Map<String,Object> params = (Map)config.getOrDefault("params", Collections.emptyMap());
```

Comment:
Should we add a check to the new trigger methods to check that they are executed against a system LEADER?